stable/postgresql/values.yaml

```diff
@@ -65,6 +65,12 @@ securityContext:
   fsGroup: 1001
   runAsUser: 1001
 
+## Pod Service Account
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+serviceAccount:
+  enabled: false
+  # name:
```

Comment:
I'd add more information about the implications on setting this parameter `serviceAccount.name`.

Please update the `values-production.yaml` with these changes too

Reply:
Thanks, also done!